db/seeds.rb

```diff
@@ -174,16 +174,22 @@
 end
 
 if Account.count == 0
```

Comment:
Let's make sure that the order of records here and the enum are identical. One option is to include id: value in the seed data.